chainerx_cc/chainerx/empty.cc

```diff
@@ -0,0 +1,2 @@
+// An empty file to workaround old CMake's issue which cannot allow us
```

Comment:
Let's add a specific version number.
```suggestion
// An empty file to workaround old CMake (<3.11)'s issue which cannot allow us
```